core/appeal/service_test.go

```diff
@@ -131,33 +137,33 @@ func (s *ServiceTestSuite) TestCreate() {
 	}
 	s.Run("should return error if got error from resource service", func() {
 		expectedError := errors.New("resource service error")
-		s.mockResourceService.On("Find", mock.Anything).Return(nil, expectedError).Once()
+		s.mockResourceService.On("Find", mock.Anything, mock.Anything).Return(nil, expectedError).Once()
```

Comment:
when we are using `mocks` it is better if we don't use `mock.Anything` if it is possible. Also at the end, we could also try asserting all expected mocks with `mock.AssertExpectations(t)`